src/Model/APContact.php

```diff
@@ -119,7 +119,8 @@ private static function fetchWebfingerData(string $addr)
 	 */
 	public static function getByURL($url, $update = null)
 	{
-		if (empty($url)) {
+		if (empty($url) || Network::isUrlBlocked($url)) {
+			Logger::info('Domain is blocked', ['url' => $url]);
```

Comment:
Should we copy this in the `Contact::getByURL` as well?

Reply:
That's why I implemented the check in the `Probe` class as well. That is some lower level check.